kornia/contrib/models/structures.py

```diff
@@ -104,3 +104,18 @@ def keypoints(self) -> Tensor | None:
     def keypoints_labels(self) -> Tensor | None:
         """The keypoints labels from the `points`"""
         return self.points[1] if isinstance(self.points, tuple) else None
+
+
+@dataclass
+class DetectionResults:
```

Comment:
Can we name this ObjectDetectionResults ? I could see having confusion with FeatureDetection, FaceDetection, etc. what are the thoughts for batches with irregular number of detections? Should this class supports lists of a much simple data class too ? Just ideas for subsequent pr

Reply:
How about for this PR, I remove the DetectionResults, and directly return the class id, scores, and bounding boxes from the model? A future PR will tackle the issue of return format.

Reply:
Agree